src/main/scala/com/campudus/tableaux/database/domain/table.scala

```diff
@@ -25,23 +25,38 @@ case object SettingsTable extends TableType {
   override val NAME = "settings"
 }
 
-case class Table(id: TableId, name: String, hidden: Boolean, langtags: Option[Seq[String]], displayInfos: Seq[DisplayInfo], tableType: TableType, tableGroup: Option[TableGroup]) extends DomainObject {
+case class Table(id: TableId,
+                 name: String,
+                 hidden: Boolean,
+                 langtags: Option[Seq[String]],
+                 displayInfos: Seq[DisplayInfo],
+                 tableType: TableType,
+                 tableGroup: Option[TableGroup])
+    extends DomainObject {
   override def getJson: JsonObject = {
-    val result = Json.obj(
-      "id" -> id,
-      "name" -> name,
-      "hidden" -> hidden,
-      "displayName" -> Json.obj(),
-      "description" -> Json.obj()
-    )
+    val result = Json.obj("id" -> id,
+                          "name" -> name,
+                          "hidden" -> hidden,
+                          "displayName" -> Json.obj(),
+                          "description" -> Json.obj())
 
     if (langtags.isDefined) {
       result.mergeIn(Json.obj("langtags" -> langtags.orNull))
     }
 
     displayInfos.foreach { di =>
-      di.optionalName.map(name => result.mergeIn(Json.obj("displayName" -> result.getJsonObject("displayName").mergeIn(Json.obj(di.langtag -> name)))))
-      di.optionalDescription.map(desc => result.mergeIn(Json.obj("description" -> result.getJsonObject("description").mergeIn(Json.obj(di.langtag -> desc)))))
+      di.optionalName.map(
+        name =>
```

Comment:
This looks strange. On other occasions, we had `.map({ ... =>`  in one line - so not having a `{...}`-block means "use more lines", which feels wrong to me.